### DISCUSSION TITLE
Mention which PR type should we include in CONTRIBUTING_BASIC.md

### DISCUSSION THREAD

**Discussion by himanshubalani** [monkeytypegame/monkeytype#4625] [Ideas] [answered]
I added a few quotes to contribute to the quotes. #4624 

I followed CONTRIBUTING_BASIC.md and opened a pull request. But to my dismay, the one check failed because a PR type must be mentioned in the PR title, which I didn't know. 

Can we update the contributing docs with the suitable PR type for addind/improving quotes?

I surfed previous PRs and "impr(quotes)" seems like a good fit for this.

Can I make this contribution in the CONTRIBUTING_BASIC.md?

**Comment by ridemountainpig** [comment, marked as answer]
Available types:
 - build
 - chore
 - ci
 - docs
 - feat
 - impr
 - fix
 - perf
 - refactor
 - revert
 - style
 - test
 
You can select the appropriate type for your pull request.
